rust/build.rs

```diff
@@ -41,47 +42,51 @@ fn install_generated_module(module_name: &str, prebuilt_var: &str, pdl_name: &Pa
         Err(_) => PathBuf::from(module_name),
     };
 
-    if Path::new(module_prebuilt.as_os_str()).exists() {
-        let out_dir = PathBuf::from(env::var("OUT_DIR").unwrap());
-        std::fs::copy(
-            module_prebuilt.as_os_str().to_str().unwrap(),
-            out_dir.join(module_name).as_os_str().to_str().unwrap(),
-        )
-        .unwrap();
+    let out_path = PathBuf::from(env::var("OUT_DIR").unwrap()).join(module_name);
+
+    if module_prebuilt.exists() {
+        std::fs::copy(module_prebuilt, &out_path).unwrap();
     } else {
-        generate_module(pdl_name);
+        generate_module(pdl_name, &out_path);
     }
+
+    // set var for use in `include!`
+    println!("cargo:rustc-env={}={}", prebuilt_var, out_path.to_str().unwrap());
 }
 
-fn generate_module(in_file: &PathBuf) {
-    let out_dir = PathBuf::from(env::var("OUT_DIR").unwrap());
-    let out_file =
-        File::create(out_dir.join(in_file.file_name().unwrap()).with_extension("rs")).unwrap();
+fn generate_module(in_file: &Path, out_path: &Path) {
+    let mut out_file = File::create(out_path).unwrap();
 
     // Find the pdl tool. Expecting it at CARGO_HOME/bin
-    let pdl = match env::var("CARGO_HOME") {
+    let pdlc = match env::var("CARGO_HOME") {
```

Comment:
Is it even desirable to maintain this code path? If someone is using Cargo vs the Blaze build, when would they want to use the external `pdlc`? Or perhaps should it be present to allow hypothetical debugging of issues specific to a `pdlc` binary, but only used if an env var is set?